docs/reference/ilm/error-handling.asciidoc

```diff
@@ -211,7 +211,11 @@ The index name must match the regex pattern `^.*-\d+` for the rollover action to
 The most common problem is that the index name does not contain trailing digits.
 For example, `my-index` does not match the pattern requirement.
 
-Append a numeric value to the index name, for example `my-index-000001`.
+The preferred solution is using data streams so bootstraping the initial time series index is not needed
```

Comment:
```suggestion
The preferred solution is to use <<data-streams,data streams>> so bootstrapping the initial time series index is not needed.
```
It may be nice to link to the data stream docs if we present this as the preferred solution.